site/src/api/types.ts

```diff
@@ -14,3 +14,19 @@ export interface ReconnectingPTYRequest {
 export type WorkspaceBuildTransition = "start" | "stop" | "delete"
 
 export type Message = { message: string }
+
+// Keep up to date with coder/codersdk/features.go
+export enum FeatureNames {
+  AuditLog = "audit_log",
+  UserLimit = "user_limit",
+  BrowserOnly = "browser_only",
+  SCIM = "scim",
+  TemplateRBAC = "template_rbac",
+  HighAvailability = "high_availability",
+  Appearance = "appearance",
+}
```

Comment:
I thought these would be auto-generated 🤔

Reply:
They are, it looks like this was my mistake when rebasing against Ammar's changes.

Reply:
Done!